SUPPORTED_PLATFORMS.md

```diff
@@ -0,0 +1,90 @@
+# Supported Platforms
+
+This document outlines the platform support matrix for different versions of the Microsoft build of Go.
+
+## Support Policy
+
+The Microsoft build of Go follows the upstream Go [Release Policy](https://go.dev/doc/devel/release#policy). This means we support each major release (1.X) until there are two newer major releases. A new Go major version is [released every six months](https://github.com/golang/go/wiki/Go-Release-Cycle), so each Go major version is supported for about one year.
```

Comment:
Partial duplicate of information in README.md? I think sticking to platforms in this document makes more sense.